test/langtools/tools/javac/T8071847/T8071847.java

```diff
@@ -83,9 +83,9 @@ byte[] hexToByte(String str) {
 
     File writeHexFile(String classFileName, String hexString) throws IOException {
         File f = new File(classFileName);
-        FileOutputStream output  = new FileOutputStream(f);
-        output.write(hexToByte(hexString));
-        output.close();
+        try (FileOutputStream output  = new FileOutputStream(f)) {
```

Comment:
```suggestion
        try (FileOutputStream output = new FileOutputStream(f)) {
```

Reply:
Thanks - should be fixed in 40a31c3bc13.